pkg/errlog/errlog.go

```diff
@@ -0,0 +1,114 @@
+package errlog
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+
+	"cloud.google.com/go/errorreporting"
+	"google.golang.org/api/option"
+)
+
+// Client holds the newly generated error logging client, so that it can be used for our own, more simple custom logging functions instead of the ones the Google provides
+type Client struct {
```

Comment:
Definitely. I think I also might rename the package to something like `gruveelogger`.

Reply:
Definitely. And as decided in the stream, I'll change the name to `logging`, because I'm pretty sure `log` and `logger` are in the VSCode extension, which will give us a lot of trouble if the names are the same.

Reply:
Definitely, makes it a little bit more general